cli/azd/pkg/pipeline/pipeline_manager.go

```diff
@@ -30,8 +31,9 @@ import (
 type PipelineAuthType string
 
 const (
-	AuthTypeFederated         PipelineAuthType = "federated"
-	AuthTypeClientCredentials PipelineAuthType = "client-credentials"
+	AuthTypeFederated               PipelineAuthType = "federated"
+	AuthTypeClientCredentials       PipelineAuthType = "client-credentials"
+	AzurePipelineClientIdEnvVarName string           = "AZURE_PIPELINE_CLIENT_ID"
```

Comment:
Can we also output the AD App IDs?  It is helpful to have that info and would be nicer to go to .env for it rather than having to try to find it in the portal.

Reply:
The id from the portal should not work..   we can only get the name from the portal, and then azd will fetch the princiapl id from ms-graph based on the name